docs/adr/0007-alerting-setup.md

```diff
@@ -0,0 +1,49 @@
+# Alerting setup for Operate First monitoring
+
+* Status: accepted
+* Deciders: hemajv, 4n4nd, anishasthana, HumairAK, tumido, mhild
+
+Technical Story: [issue-1](https://github.com/operate-first/SRE/issues/14), [issue-2](https://github.com/operate-first/SRE/issues/19), [issue-3](https://github.com/operate-first/blueprint/issues/16)
+
+## Context and Problem Statement
+
+As we have multiple services deployed and monitored in the Operate First environment (ex. Jupyterhub, Argo, Superset, Observatorium, etc), we need to implement an incident reporting setup for handling outages/incidents related to the services.
+
+All the services are being monitored by [Prometheus](https://prometheus.io/). Prometheus scrapes and stores time series data identified by metric key/value pairs for each of the available services. These metrics can be used for measuring the service performance and alerting on any possible service degradation such as basic availability, latency, durability etc.
```

Comment:
where do we define and document the SLA/SLO/SLI for each of the services?

Reply:
they are in the [SRE repo](https://github.com/operate-first/SRE/tree/master/sli-slo), added a link pointing to it